doc/testing-your-application.rst

```diff
@@ -23,11 +23,36 @@ By having your test classes extend one of the test case classes, you get:
 The recommended way to configure the test case is by setting constants
 in your ``phpunit.xml``. Alternatively, you can override the getter methods.
 
+Web Server
+~~~~~~~~~~
+
 You will need to run a web server to provide the PHP application you want to
-test. The test cases only handle running the caching proxy. With PHP 5.4 or
-newer, the easiest is to use the PHP built in web server. See the
-``WebServerListener`` class in ``tests/Functional`` and how it is registered in
-``phpunit.xml.dist``.
+test. The test cases only handle running the caching proxy. It’s easiest to
+use PHP’s built in web server. Include the WebServerListener in your
+`phpunit.xml`:
```

Comment:
you need double backticks in rst, single ones don't work. but i agree with saying phpunit.xml instead of .dist